docs/FAQ.md

```diff
@@ -3,49 +3,102 @@ title: eScriptorium Tutorial - FAQ
+
+!!! Tip "More info" 
+    ➡️ See [About](about.md)
+
+## Who is responsible for eScriptorium?
+
+!!! Note
+    The text in this section cites eScriptorium's FAQ: 
+
+    ➡️ [https://gitlab.com/scripta/escriptorium/-/wikis/Who-is-responsible-for-eScriptorium%3F](https://gitlab.com/scripta/escriptorium/-/wikis/Who-is-responsible-for-eScriptorium%3F)
+
+> eScriptorium is an Open Source project and so it has a number of contributors and partners around the world. However, the project is hosted and lead by the Digital Humanities team at the [École Pratique des Hautes Études – Université Paris Sciences et Lettres](https://ephe.psl.eu/), in [AOROC](http://www.archeo.ens.fr/) in Paris. This core team comprises:
+> 
+> - [Daniel Stökl Ben Ezra](https://www.ephe.psl.eu/daniel-stoekl-ben-ezra) (Co-Director)
+> - [Peter Stokes](https://www.ephe.psl.eu/peter-stokes) (Co-Director)
+> - Benjamin Kiessling (machine vision engineer, responsible for [Kraken](https://kraken.re/))
+> - Robin Tissot (lead project engineer for eScriptorium)
+> 
+> In addition to this are several other contributors, including a team from [Teklia](https://teklia.com/), and from the [Open Islamicate Texts Initiative](https://openiti.org/) at the University of Maryland and Northeastern University, among others.
+>
+> If you have questions or need help then you can try contacting us (follow the links for the names above to find contact details), but a more effective means might be to join [our community on Gitter](https://gitter.im/escripta/escriptorium) and ask your questions there.
+>
+> Finally, if you want to contact the eScriptorium team then please do not use the 'Contact' form on the website, as this goes to the local admins for the site and not to us (see [I contacted you via the eScriptorium website: why didn't you reply?](https://gitlab.com/scripta/escriptorium/-/wikis/I-contacted-you-via-the-eScriptorium-website:-why-didn't-you-reply%3F)).
 
 ## Is eScriptorium accessible offline?
 
+If you use an instance of eScriptorium hosted on a web server, no, you cannot access eScriptorium offline.
+
+If you use a [local deployment](https://gitlab.com/scripta/escriptorium/-/wikis/docker-install) of eScriptorium (installed on your machine and hosted by a virtual server), yes, you can access eScriptorium offline.
+
 ## Can I synchronize a local eScriptorium application with a server?
 
+eScriptorium is a decentralized software, which means that there are as many databases as there are instances of eScriptorium. It is impossible to synchronise two instances, unless they share the same database, which is unlikely.
+
+You can, however, update the content of your projects and documents from one instance to another with the help of the [export](walkthrough_export.md) and [import](walkthrough_import.md) features. For this, you need to have an account on both the source and the target instances.
```

Comment:
Maybe it is just me, but I find it weird that you wrote export and import, instead of import and export, feels unnatural (but it might be just me) !

Reply:
I see, I wrote it in this order because in this case you want to export the data from an instance and then import it into another instance